include/alp/reference/blas1.hpp

```diff
@@ -1332,19 +1336,37 @@ namespace alp {
 	RC foldl( Vector< IOType, IOStructure, Density::Dense, IOView, IOImfR, IOImfC, reference > & x,
```

Comment:
Multiline function parameter list should be broken as:
```
RC function(
    type1 var1,
    type2 var2
) {
    // code
}
```
Please fix in all instances